core/src/test/java/org/apache/calcite/test/SqlXmlFunctionsTest.java

```diff
@@ -35,6 +37,10 @@
  */
 public class SqlXmlFunctionsTest {
 
+  @BeforeAll public static void setUp() {
```

Comment:
Could not find a better way to fix the test case. Explored LocaleEnUs but no luck